tests/10-verify.spec.js

```diff
@@ -1,87 +1,58 @@
-chai.should();
+const MultiLoader = require('./MultiLoader');
+const realContexts = require('../lib/contexts');
+const invalidContexts = require('./contexts');
+const credential = require('./mocks/credential');
+const assertionController = require('./mocks/assertionController');
 
-// const testContextLoader = () => {
-//   // FIXME: used credentials-context module when available
-//   const contexts = new Map([[
-//     constants.CREDENTIALS_CONTEXT_URL,
-//     require('./contexts/credentials-v1.jsonld')
-//   ]]);
-//   return async url => {
-//     if(!contexts.has(url)) {
-//       throw new Error('NotFoundError');
-//     }
-//     return {
-//       contextUrl: null,
-//       document: jsonld.clone(contexts.get(url)),
-//       documentUrl: url
-//     };
-//   };
-// };
-//
-// // documents are added to this documentLoader incrementally
-// const testLoader = new MultiLoader({
-//   documentLoader: [
-//     // CREDENTIALS_CONTEXT_URL
-//     testContextLoader(),
-//     // DID_CONTEXT_URL and VERES_ONE_CONTEXT_URL
-//     v1DocumentLoader
-//   ]
-// });
+chai.should();
+const {expect} = chai;
 
-const contexts = require('../lib/contexts');
+const contexts = Object.assign({}, realContexts);
 
-async function documentLoader(url) {
-  const context = contexts[url];
-  if(context) {
+const testContextLoader = () => {
+  // FIXME: used credentials-context module when available
+  for(const key in invalidContexts) {
+    const {url, value} = invalidContexts[key];
+    contexts[url] = value;
+  }
+  return async url => {
+    if(!contexts[url]) {
+      throw new Error('NotFoundError');
+    }
     return {
       contextUrl: null,
-      documentUrl: url,
-      document: context
+      document: jsonld.clone(contexts[url]),
+      documentUrl: url
     };
-  }
-  throw new Error(`${url} is not an authorized supported context url.`)
-}
+  };
+};
 
-const assertionController  = {
-  '@context': 'https://w3id.org/security/v2',
-  id: 'https://example.com/i/carol',
-  assertionMethod: [
-    // actual key is going to be added in the before() block below
+// documents are added to this documentLoader incrementally
+const testLoader = new MultiLoader({
+  documentLoader: [
+    // CREDENTIALS_CONTEXT_URL
+    testContextLoader(),
   ]
-};
+});
 
-let suite, keyPair, verifiedCredential;
+const documentLoader = testLoader.documentLoader.bind(testLoader);
 
-const credential = {
-  "@context": [
-    "https://www.w3.org/2018/credentials/v1",
-    "https://www.w3.org/2018/credentials/examples/v1"
-  ],
-  "id": "http://example.edu/credentials/1872",
-  "type": ["VerifiableCredential", "AlumniCredential"],
-  "issuer": "https://example.edu/issuers/565049",
-  "issuanceDate": "2010-01-01T19:23:24Z",
-  "credentialSubject": {
-    "id": "did:example:ebfeb1f712ebc6f1c276e12ec21",
-    "alumniOf": "<span lang='en'>Example University</span>"
-  }
-}
+let suite, keyPair, verifiedCredential;
 
-before(async () => {
+before(async function() {
```

Comment:
Why the switch here from `() =>` to `function()` ?

Reply:
https://mochajs.org/#arrow-functions

I do not use arrow functions in mocha at all.

Reply:
No time like the present to start using em :) 
Change it back to an arrow function, if you don't mind.

Reply:
you do understand it means that if in a future test you need access to the mocha api it will throw an error?

Reply:
I do, yes. I can't think of a single place, among all the digital bazaar's repos, that we would need to access the mocha API. 
And if that case comes up, the developer writing it can change it then.

Reply:
I can think of tons including custom timeouts on integration tests that might depend on prolonged api calls, the ability to delay tests, ignore non fatal errors etc. anyways you are busy so I made the switch, but from previous experience using arrow functions in mocha will lead to pain in the future.